src/main/java/Duke.java

```diff
@@ -1,10 +1,140 @@
+import java.util.Scanner;
+
 public class Duke {
+    static int taskCount = 0;
+    static Task[] taskList = new Task[100];
+    static final String COMMANDS = "Commands:\n    todo taskName\n    deadline deadlineName /by time\n" 
+            + "    event eventName /at time\n    list\n    done taskNumber\n    help\n    bye\n";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+        displayWelcomeMessage();
+        inputAndExecuteCommand();
+        displayExitMessage();
+    }
+
+    private static void displayExitMessage() {
+        String exitMessage = "Sad to see you go! ): See you soon!";
+        printWithBorder(exitMessage);
+    }
+
+    private static void inputAndExecuteCommand() {
+        String line;
+        Scanner scanner = new Scanner(System.in);
+        
+        while (true) {
+            line = scanner.nextLine();
+            String[] commandTypeAndArg = line.split(" ", 2);
+            String commandType = commandTypeAndArg[0];
+            String commandArg = null;
```

Comment:
Avoid assigning null value to variable

Reply:
thank you! i will replace this with an empty string